src/Mod/Fem/femsolver/ratel/ratel_util.py

```diff
@@ -0,0 +1,30 @@
+import subprocess
+from platform import system
+
+def get_location(name):
+    if system() in ("Linux", "Darwin"):
+        command = "find / -type d -name '" + name + "' -exec test -d \"{}/.git\" \\; -print 2>/dev/null"
```

Comment:
This is expensive and non-deterministic. The user may have multiple of these directories and it may produce a different incompatible set each time.